test/fixtures/frontend.yml

```diff
@@ -37,10 +25,9 @@ build:
     CGO_ENABLED: "0"
     GOGC: off
     GOFLAGS: -trimpath
-    GOPATH: /go
+    # GOPATH: /go
```

Comment:
Is this intended to be a comment?

Reply:
Hmm, I think I changed this for debugging purposes and never changed it back.

Reply:
Fixed this.